src/reducers/bottom-bar.js

```diff
@@ -0,0 +1,28 @@
+//@flow
+export const types = {
+  UPDATE_BOTTOM_BAR_SIZE: 'top-bar/UPDATE_BOTTOM_BAR_SIZE'
```

Comment:
1. I would add this action to the shell. it's already holding all sizes (gui, video etc.)
2. change to UPDATE_BOTTOM_BAR_**HEIGHT**